Documentation/configuration/sctp.rst

```diff
@@ -37,6 +37,7 @@ Cilium supports basic SCTP support. Specifically, the following is supported:
    Cilium does not support the following for SCTP:
     - Multihoming
     - Policies for pod-to-VIP
-    - KPR
+    - :ref:`Kube-proxy replacement (KPR)<kubeproxy-free>`, except for the 
+      specific cases listed above (e.g., NodePort Service is not supported with KPR).
```

Comment:
I find the double negation confusing (“it doesn't work... except for the cases listed above”; and it's not super-clear whether “above” refers to the note or to the list above (I understand it's the list above, but I had to read this a few times).

How about something like:

```rst
    - :ref:`Kube-proxy replacement (KPR)<kubeproxy-free>` when port rewriting
      is necessary: for example, NodePort Services are not supported with the
      combination of KPR and SCTP.
```